packages/property-view/README.md

```diff
@@ -0,0 +1,44 @@
+<div align='center'>
+
+<br />
+
+<img src='https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/theia.svg?sanitize=true' alt='theia-ext-logo' width='100px' />
+
+<h2>ECLIPSE THEIA - PROPERTY-VIEW EXTENSION</h2>
+
+<hr />
+
+</div>
+
+## Description
+
+The `@theia/property-view` extension contributes a generic, global property view based on Theia's global selection.
+
+The property view widget can be opened/toggled either via menu _View->Properties_ or via shortcut <kbd>Shift+Alt+P</kbd>. It is located in the bottom dock area per default.
+
+The following two default content widgets are implemented in this extension:
+- EmptyPropertyViewWidget: If no other widget can be provided, a simple message (_No properties available_) is shown.
+- ResourcePropertyViewWidget: Can handle file and monaco editor selections and displays the general `FileStat` information (e.g. location, name, last modified) in a TreeWidget.
```

Comment:
Why is it `monaco` specific?

https://github.com/eclipse-theia/theia/blob/42042974c59f148974de5e8fd90cba7222cee82f/packages/editor/src/browser/editor.ts#L200

Reply:
`Navigatable` looks even more generic: https://github.com/eclipse-theia/theia/blob/42042974c59f148974de5e8fd90cba7222cee82f/packages/core/src/browser/navigatable.ts#L29

Reply:
You're totally right, thank you for spotting that! :100: 
I make use now of the `Navigatable` interface for the selection check and URI retrieval and could therefore optimize the dependencies and remove `@theia/monaco` completely from this extension. :+1: